mail_restrict_follower_selection/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Therp BV <http://therp.nl>.
+#    Copyright (C) 2017 Komit <http://www.komit-consulting.com>.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Restrict follower selection",
+    "version": "11.0.1.0.0",
+    "author": "Komit, "
```

Comment:
I don't think that Komit is not the author as Komit only takes care of the migration.

For a migration, I think that Komit should only be added as a contributor, in the README.rst

Reply:
yes, author stays Therp BV and the OCA